js/main.js

```diff
@@ -0,0 +1,63 @@
+const domBookList = document.querySelector('#book-list');
+const inputTitle = document.querySelector('#input-title');
+const inputAuthor = document.querySelector('#input-author');
+const buttonAdd = document.querySelector('#button-add');
+
+const stringBookList = localStorage.getItem('bookList');
+
+let bookList = [];
+
+if (stringBookList) {
+  const parsedBookList = JSON.parse(stringBookList);
+  if (Array.isArray(parsedBookList)) {
+    bookList = parsedBookList;
+  }
+}
+
+function removeBook(index) {
+  bookList.splice(index, 1);
+//   drawBooks();
```

Comment:
- [ ] Kindly remove commented code.

Reply:
Thanks very much, we forgot to remove that line.